### ISSUE TITLE
About your SegFormer

### ISSUE THREAD

**Issue by redorangeyellowy** [MengHao666/HDR#4]
Thank you for the good research.

According to the paper, you said you pre-train SegFormer, but did you use only two hand images except one hand image on the Interhand 2.6M dataset?

**Comment by MengHao666** [comment]
> Thank you for the good research.
> 
> According to the paper, you said you pre-train SegFormer, but did you use only two hand images except one hand image on the Interhand 2.6M dataset?

Sure, we only use two hand images to train segFormer.  The Interhand 2.6M dataset has been divided into single hand sequences and interacting hand sequence.

**Comment by redorangeyellowy** [comment]
Thank you for your reply.

As far as I know, pixel-level segmentation mask images are required to train SegFormer, but the InterHand2.6M dataset basically does not provide them.

May I ask how you got it?

**Comment by MengHao666** [comment]
> 

InterHand2.6M  has provided optimized MANO hand model parameters. You can use render code of its repo to project mesh into image plane. Thus, groundtruth segmentation mask could be obtained.

**Comment by redorangeyellowy** [comment]
Okay. Thank you for your reply.
